pkgs/development/arduino/platformio/chrootenv.nix

```diff
@@ -25,6 +25,7 @@ let
       setuptools
       pip
       bottle
+      git
```

Comment:
`git` is not in `python.pkgs`

Reply:
ya i put in wrong place i repushed it correctly